index.js

```diff
@@ -1,4 +1,25 @@
 'use strict';
+
+const parseMessageAttributes = input => {
+	const result = {};
+
+	for (const key of Object.keys(input)) {
+		const value = input[key];
+
+		let resultValue;
+
+		try {
+			resultValue = JSON.parse(value.Value);
```

Comment:
Because values are always passed as a string, this is done here to parse numbers, arrays and booleans.